okteto.yml

```diff
@@ -0,0 +1,27 @@
+build:
+  guestbook:
+    image: okteto.dev/guestbook:1.0.0
```

Comment:
dont set the `image`, we will use `OKTETO_BUILD_GUESTBOOK_IMAGE` to let customize use the right image tag